travisBuildScript.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+set -e
+
+mkdir $BUILD_DIR
+cd $BUILD_DIR
+
+CMAKE_FLAGS="-DCMAKE_CXX_COMPILER=$COMPILER"
```

Comment:
Do you need to also pass the `COMPILER_FLAGS` variable or does cmake fetch it from `env`?